### DISCUSSION TITLE
Any way to add an action into the topbar without overwriting views?

### DISCUSSION THREAD

**Discussion by StaffCollab** [filamentphp/filament#11680] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I want to put a chat icon that will open a chat pop-up in the top bar right next to the notifications icon, like this
![Screenshot 2024-03-04 103601](https://github.com/filamentphp/filament/assets/111918553/65993542-20f6-4569-8d48-c2b13d27ee75)
Is there any why to do this without tempering with the default views?

**Comment by goyne** [comment]
I managed to add a button on that position using the renderHook panels::global-search.after.

**Comment by leandrocfe** [comment]
[Render hooks](https://filamentphp.com/docs/3.x/panels/configuration#render-hooks)
[Available hooks](https://filamentphp.com/docs/3.x/support/render-hooks#available-render-hooks)
